h/auth/__init__.py

```diff
@@ -13,33 +13,15 @@
 from h.auth.util import default_authority, groupfinder
 from h.security import derive_key
 
-__all__ = ("DEFAULT_POLICY", "WEBSOCKET_POLICY")
+__all__ = ("TOKEN_POLICY",)
 
 log = logging.getLogger(__name__)
 
-PROXY_POLICY = RemoteUserAuthenticationPolicy(
-    environ_key="HTTP_X_FORWARDED_USER", callback=groupfinder
-)
-TICKET_POLICY = pyramid_authsanity.AuthServicePolicy()
-
+# We export this for the websocket to use as it's main policy
 TOKEN_POLICY = TokenAuthenticationPolicy(callback=groupfinder)
-AUTH_CLIENT_POLICY = AuthClientPolicy()
-
-API_POLICY = APIAuthenticationPolicy(
-    user_policy=TOKEN_POLICY, client_policy=AUTH_CLIENT_POLICY
-)
-
-DEFAULT_POLICY = AuthenticationPolicy(
-    api_policy=API_POLICY, fallback_policy=TICKET_POLICY
-)
-WEBSOCKET_POLICY = TOKEN_POLICY
 
 
 def includeme(config):
-    # pylint: disable=global-statement
-    global DEFAULT_POLICY
-    global WEBSOCKET_POLICY
```

Comment:
This is kind of totally to me for two reasons:

* We declare a global `WEBSOCKET_POLICY` so we can change the value (??) but we then set it to the same value (`TOKEN_POLICY`)
* We declare a global `DEFAULT_POLICY` for the same reason, but nobody imports it